NotificationBox/Kwc/Component.defer.js

```diff
@@ -0,0 +1,44 @@
+"use strict";
+var $ = require('jquery');
+var onReady = require('kwf/on-ready');
+
+function setCookieValue(key, value) {
```

Comment:
Not in global namespace, move it into onReady.onRender(... like
`var setCookieValue = function...`